createExeDir.bat

```diff
@@ -0,0 +1,17 @@
+@echo off
+py -m pip install --upgrade -r requirements.txt
+py -m pip install pyinstaller
+
+pyinstaller --noconfirm --onefile --console --hidden-import "pkg_resources.py2_warn" --hidden-import "Flask" --hidden-import "PyYAML" --hidden-import "pywin32"  .\service.py
+
+mkdir ServicePackage
+
+COPY .\dist\service.exe /B .\ServicePackage
+mkdir .\ServicePackage\mockoon_files
+COPY .\mockoon_files .\ServicePackage\mockoon_files
+copy createServiceFromExe.bat /B .\ServicePackage
+copy config.yaml /A .\ServicePackage
+
+RMDIR/Q/S build
+RMDIR/Q/S dist
+DEL service.spec
```

Comment:
wydaje mi się że zostawienie wszystkiego w katalogu `dist` będzie bardziej intuicyjne, do tego katalogu dorzucił bym potrzebne pliki i tak to zostawiał

dodatkowe pliki i katalogi, które powstały podczas budowania dodał bym do `.gitignore`

sprawdził bym czy `pyinstaler` czyści te pliki i katalogi podczas kolejnego budowania, jak jeżeli nie to przed budowaniem zadbał bym o to żeby "posprzątać"

Reply:
Te dodatkowe pliki i katalogi są tak czy siak tymczasowe i auto-py-to-exe usuwa je od razu, więc nie wiem czemu mielibyśmy je trzymać. A przerzucanie do ServicePackage jest według mnie po prostu ładniejsze ale jak chcemy to może zostawiać w dist